### PR TITLE
Setup code, build, package.json, tests, readme

### PR DESCRIPTION
**Pull request by xamfoo**
Also updated dependencies.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+/**
+ * Creates a higher-order reducer which map actions before passing to a reducer
+ * Specify a list of action types to whitelist actions for mapping.
+ *
+ * mapAction(
+ *   actionMapper: ?((action: Object) => newAction: Object) = (x) => x,
+ *   actionTypes: ?Array<string>
+ * ): (reducer) => reducer
+ */
+const mapAction = (actionMapper = identity, actionTypes = null) => {
```
Comment (belkosyak) [inline src/index.js]: I think `actionMapper` should be required, because this HOR useless without it :)

File: test/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+const spies = {
+  reducer: jasmine.createSpy('reducer')
+};
+
+const teardown = () => {
+  for (const name in spies) {
+    spies[name].calls.reset();
+  }
+}
```
Comment (belkosyak) [inline test/index.spec.js]: put comma please

File: test/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+const should = {
+  callReducer: (params) => expect(spies.reducer).toHaveBeenCalledWith(...params)
+}
```
Comment (belkosyak) [inline test/index.spec.js]: put comma please

**Comment by xamfoo** [comment]
Fixed.

**Pull request merged**